modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/RightSidebar/RightSidebarObjectDefinitionDetails.tsx

```diff
@@ -142,7 +142,7 @@ export function RightSidebarObjectDefinitionDetails({
 
 		makeFetch();
 		// eslint-disable-next-line react-hooks/exhaustive-deps
-	}, []);
+	}, [selectedObjectDefinitionNode?.id]);
```

Comment:
hey @igor-franca! did you verify if the onBlur of the right sidebar will still work after this change? i'm asking because we know the [  ] means it will only execute on the first render, and here will be on more time (and who made this way was daniel during his onBlur story). can you confirm this for me?

Reply:
Hey @rebsilva, I attached a video showing this bahavior.